### PR TITLE
Overview moved and modified

### PR DESCRIPTION
**Pull request by julieraines**
. . . from original request per Scott's helpful link (which gave me a slightly different result than when I did the pull request method - difference was patch-1 in head fork as opposed to master)

### CODE REVIEW

File: docs/docs/Resources/troubleshooting.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Once you're setting up your loop, you may also want to oref0-reset-usb (`oref0-r
 
 OpenAPS uses git as the logging mechanism, so it commits report changes on each report invoke. Sometimes, due to "unexpected" power-offs (battery dying, unplugging, etc.),the git repository gets broken. When it happens you will receive exceptions when running any report from openaps. As git logging is a safety/security measure, there is no way of disabling these commits.
 
-You may see an error that references a loose object, or a corrupted git repository. To fix a corrupted git repository you can run `oref0-fix-git-corruption.sh`, it will try to fix the repository, and in case when repository is definitely broken it copies the remainings in a safe place (`tmp`) and initializes a new git repo.
+You may see an error that references a loose object, or a corrupted git repository. To fix a corrupted git repository you can run `oref0-fix-git-corruption`, it will try to fix the repository, and in case when repository is definitely broken it copies the remainings in a safe place (`tmp`) and initializes a new git repo.
```
Comment (sspade) [inline docs/docs/Resources/troubleshooting.md]: Is the copy moved into "/tmp" or a tmp directory under the current openaps directory?

Comment (scottleibrand) [inline docs/docs/Resources/troubleshooting.md]: /tmp by default: https://github.com/openaps/oref0/blob/dev/bin/oref0-reset-git.sh#L34

Comment (sspade) [inline docs/docs/Resources/troubleshooting.md]: So this dir will be wiped daily then right? 
```
21:59:00 $ grep tmp /etc/defaults/periodic.conf
110.clean-tmps
daily_clean_tmps_enable="YES"				# Delete stuff daily
```
I'll edit and note this in the docs, post merge of 311 (this pr.)

**Comment by scottleibrand** [comment]
Just need you to merge https://github.com/julieraines/docs/pull/1 and then this should be good to merge to dev.

File: docs/docs/walkthrough/phase-1/openaps-to-nightscout.md [hunk 2]
```diff
@@ -6,9 +6,10 @@ To view your data on your Nightscout site, start by doing the following:
 
 So this would be your actual https://myname.azurewebsites.net  or https://myname.herokuapp.com (without the carrots of course). Your API_SECRET is listed in your Azure or Heroku settings.
 
-To test this: `openaps use ns shell preflight`
+To test this: `openaps use ns shell preflight`\
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-1/openaps-to-nightscout.md]: Is this trailing \ intentional?

**Pull request merged**